### ISSUE TITLE
Questions on training models with different compression levels

### ISSUE THREAD

**Issue by 1747933957** [ttlzfhy/D-DPCC#8]
I want to train D-DPCC models on my own dataset, but I don't know how to control the degree of compression. Which parameters should I adjust during training to control the degree of compression? Or should I modify the parameters during the testing phase to control the degree of compression?

**Comment by ttlzfhy** [comment]
Just change the 'lamb' parameter in trainer.py. Loss function of DDPCC is L=R+lambda*D, in which R is the bitrate and D is the BCE  distortion, larger lambda makes compression ratio smaller, bitrate higher and distortion lower. You can look for the paper for more detail.
